packages/eslint-plugin-stories/src/index.ts

```diff
@@ -1,14 +1,17 @@
+import noExtResourcesInStories from './rules/no-ext-resources-in-stories';
 import noJestInStories from './rules/no-jest-in-stories';
 import noTopLevelStoryArgs from './rules/no-top-level-story-args';
 import storiesDefaultExport from './rules/stories-default-export';
 
 const rules = {
+  'no-ext-resources-in-stories': noExtResourcesInStories,
```

Comment:
Can you add an entry for this rule to https://github.com/chanzuckerberg/frontend-libs/blob/75934261018a48439f5552a02fd32b512bd2e3fd/packages/eslint-plugin-stories/README.md#rules ?

Reply:
thanks for the reminder!